### ISSUE TITLE
Version 0.33.0 changes to tun causes failure to spin up VMs. 

### ISSUE THREAD

**Issue by birdiesanders** [kubevirt/kubevirt#4221]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind enhancement

**What happened**:
When standing up a new cluster, I was unable to get any VM to boot. Upon investigation, all the VMs were stuck after being unable to attach tun0 to the pods. 
**What you expected to happen**:
I expected the VMs to spin up as they did with 0.32.0
**How to reproduce it (as minimally and precisely as possible)**:
We are using Nokia DANM for our network layer, so the basic steps are as follow;
Install and configure Nokia DANM, install kubevirt following the install section of the manual, install CDI with same, run the kubectl create to spin up a VM. 

**Anything else we need to know?**:
I ripped 0.33.0 back out, and replaced it with 0.32.0 and everything works as expected, without any further issues. 

**Environment**:
- KubeVirt version (use `virtctl version`): Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.1", GitCommit:"206bcadf021e76c27513500ca24182692aabd17e", GitTreeState:"clean", BuildDate:"2020-09-09T11:26:42Z", GoVersion:"go1.15", Compiler:"gc", Platform:"linux/amd64"}

- Kubernetes version (use `kubectl version`): Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.1", GitCommit:"206bcadf021e76c27513500ca24182692aabd17e", GitTreeState:"clean", BuildDate:"2020-09-09T11:18:22Z", GoVersion:"go1.15", Compiler:"gc", Platform:"linux/amd64"}

- VM or VMI specifications: Ubuntu Focal setup with CDI, from the standard cloud-image. 

- Cloud provider or hardware configuration: OCP compute nodes, second generation. 

- OS (e.g. from /etc/os-release): Ubuntu 20.04

- Kernel (e.g. `uname -a`): Linux host-250 5.4.0-47-generic #51-Ubuntu SMP Fri Sep 4 19:50:52 UTC 2020 
x86_64 x86_64 x86_64 GNU/Linux

- Install tools: kubectl
- Others:

**Comment by rmohr** [comment]
> When standing up a new cluster, I was unable to get any VM to boot. Upon investigation, all the VMs were stuck after being unable to attach tun0 to the pods.

Hm, that is tricky. Could you share virt-handler logs and a logs from one of the virt-launcher pods which are stuck? I can only say that things are working four our kubeadm clusters (1.16, 1.17, 1.18 and 1.19 with selinux) without more logs.

**Comment by rmohr** [comment]
@jean-edouard @maiqueb would be great if you could have a look if we can get some logs.

**Comment by jean-edouard** [comment]
https://github.com/kubevirt/kubevirt/pull/4232

**Comment by birdiesanders** [comment]
I will attempt to find logs, but I may have to build an entire cluster in
order to replicate this, so please bear with me if the lead time is a bit
of time. We have some spare OCP compute nodes, so I should be able to pull
this off by the end of the week. I’ll probably enlist one of the other
engineers as well to speed up the process a bit.

**Comment by rmohr** [comment]
@birdiesanders #4232 is merged on master, you can instead also try out the latest developer build. It if works for you then, we don't need anything in addition.

Here how to deploy the developer build: https://kubevirt.io/user-guide/#/installation/installation?id=installing-the-daily-developer-builds

**Comment by birdiesanders** [comment]
Ok, this current testing cluster has some test customers running on it until the end of the week. After that, we can start doing potentially destructive things such as upgrading kubevirt. The plan as of now is to let them have the cluster through the weekend, and on Monday we will start testing a few changes that we have come up with, and I will include the update to the newer version in that list and report back to you with the results. Thank you for your help, and your patience. You guys have done incredible work with this project though, I am constantly impressed with the project and just how reliable it is.

**Comment by birdiesanders** [comment]
Alright, I am finally in position to test this fix. We installed vanilla 0.33.0 and it still suffers tun0 failures. The dev branch will be our best bet?

**Comment by birdiesanders** [comment]
Hey gang, just wanted to let you know, the team and I finally got the time to build a new cluster, we installed the dev branch and the tun0 issue is resolved. Thanks for kicking ass, looking forward to opening more tickets when I break stuff in the future. I am going to go ahead and close this after 24 hours, as I have a question; What release will have these fixes in place? I assume this will be 0.34.0 or 0.35.0?
